tsconfig.json

```diff
@@ -5,8 +5,12 @@
     "moduleResolution": "node",
     "jsx": "react",
     "pretty": true,
+    "strict": true,
     "skipLibCheck": true,
-    "allowJs": true
+    "declaration": true,
+    "removeComments": true,
+    "emitDeclarationOnly": true,
+    "outFile": "types/index.d.ts"
```

Comment:
the `types/index.d.ts` file is generated via the `outFile` property in the `tsconfig.json`